src/App.tsx

```diff
@@ -28,11 +27,14 @@ class App extends Component {
   }
 
   async initializeServices(config: Config) {
-    if (config.content.some((el) => el.type === PersonalShelf.ContinueWatching)) {
+    // We only request favorites and continue_watching data if these features are enabled
+    if (config?.features?.continue_watching_list) {
```

Comment:
I added this check to the validation function. It requires both content item and a playlist in features.

```
  continueWatchingList: string().test('has-continue_watching-list-element', 'errorMessage', (value, context) => {
    // @ts-expect-error https://github.com/jquense/yup/issues/1631
    const { content, features } = context.from[1].value as Config;
    return checkAdditionalFeatures(content, value, PersonalShelf.ContinueWatching);
  }),
```

So it should not reach this stage if there is only content item or a playlist in the features section.

Reply:
It seems cleanest to me to only enforce that there is a playlist id set if there's a content item that needs it, but not the other way around. But that being said, this would mean we initialize the service if the playlist id is set, so this should be fine either way.

Reply:
Changed it back to content item presence